include/igl/copyleft/cgal/assign_scalar.h

```diff
@@ -51,6 +53,7 @@ namespace igl
         const CGAL::Exact_predicates_exact_constructions_kernel_with_sqrt::FT & cgal,
         float& d);
 
+#ifndef WIN32
```

Comment:
Why is this a non-windows feature? In general, we sure try to make everything available to everyone.

Reply:
See discussion in #693. Apparently `<gmpxx.h>` is not distributed with CGAL on Windows (I also tried to compile it on my side and it was complaining about this part).

Reply:
gmp is compiled with mingw on windows. Mingw c binary objects can link directly to VS binaries, but cpp objects cannot (idk why). It is thus not possible to use gmpxx directly, you will have to compile it separately with VS. I suggest to try to not use gmpxxin libigl, and stick to the standard c functions in gmp instead.